files/brews/mysql.rb

```diff
@@ -5,9 +5,9 @@
 class Mysql < Formula
 
   homepage 'http://dev.mysql.com/doc/refman/5.6/en/'
-  url 'http://s3.amazonaws.com/better-boxen/src/mysql-5.6.20.tar.gz'
-  sha1 '10f9ed2bbf3fbf227b799f1c7af3a0c7f7cf4d95'
-  version '5.6.20-boxen1'
+  url 'http://s3.amazonaws.com/better-boxen/src/mysql-5.6.21.tar.gz'
```

Comment:
so you had to download and host this yourself? what is this, source or a binary?